packages/mobile/locales/es-419/walletFlow5.json

```diff
@@ -56,6 +56,8 @@
   "escrowedPaymentReminder": "Reacuerda al receptor aceptar el pago",
   "escrowedPaymentReminderSms":
     "¡Un recordatorio amistoso de que aún no ha canjeado sus dólares de celo!",
+  "escrowedPaymentReminderSmsNoData":
+    "¡Un recordatorio amistoso de que aún no ha canjeado sus dólares de celo! ¡Descargue la aplicación móvil para comenzar!",
```

Comment:
This was done on Google translate. Is there a process to double check translations?

Reply:
¡fantástico!